docs/chapter_hashing/hash_map.md

```diff
@@ -63,19 +63,70 @@ comments: true
 === "C++"
 
     ```cpp title="hash_map.cpp"
+    /* 初始化哈希表 */
+    unordered_map<int, string> map;
 
+    /* 添加操作 */
+    // 在哈希表中添加键值对 (key, value)
+    map[12836] = "小哈";
+    map[15937] = "小啰";
+    map[16750] = "小算";
+    map[13276] = "小法";
+    map[10583] = "小鸭";
+
+    /* 查询操作 */
+    // 向哈希表输入键 key ，得到值 value
+    string name = map[15937];
+
+    /* 删除操作 */
+    // 在哈希表中删除键值对 (key, value)
+    map.erase(10583);
     ```
 
 === "Python"
 
     ```python title="hash_map.py"
-
+    """ 初始化哈希表 """
+    mapp = {}
+
+    """ 添加操作 """
+    # 在哈希表中添加键值对 (key, value)
+    mapp[12836] = "小哈"
+    mapp[15937] = "小啰"
+    mapp[16750] = "小算"
+    mapp[13276] = "小法"
+    mapp[10583] = "小鸭"
+
+    """ 查询操作 """
+    # 向哈希表输入键 key ，得到值 value
+    name = mapp[15937]
+
+    """ 删除操作 """
+    # 在哈希表中删除键值对 (key, value)
+    mapp.pop(10583)
     ```
 
 === "Go"
 
-    ```go title="hash_map.go"
-
+    ```go title="hash_map_test.go"
```

Comment:
Just keep the original `hash_map.go`.

Reply:
But in this part, there is no way to separate different hash map operations from test samples, unless the current function is directly called by the test function

Reply:
I see. My fault.